### DISCUSSION TITLE
Payload CMS training course

### DISCUSSION THREAD

**Discussion by sadeghnakhjavani** [payloadcms/payload#5446] [General]
I recently became interested in Payload CMS and am looking for a training course or a document to learn more.

Can anyone help me with this?

Here are some additional details:
 * I am a beginner and have no prior experience with the CMS.
 * I am looking for a comprehensive and easy-to-follow learning resource.
 * I would prefer a video course, but I am open to other formats as well.

Any suggestions would be greatly appreciated.

Thanks...

**Comment by Notorious-Ali** [comment]
here is a video by the founder of PayloadCms: 
https://www.youtube.com/watch?v=bxWsZTtqs80&t=4784s
and also this from a maintainer i think : 
https://www.youtube.com/watch?v=lmeOiOtvdT8

better to start by installing it for yourself and study on provided Documentation here: 
https://payloadcms.com/docs/getting-started/what-is-payload

if you want to run a ecommerce : 
https://www.youtube.com/watch?v=3JUsg-WsU9o

**Reply by sadeghnakhjavani** [reply]
Thank you...
I will check them

**Comment by jfrosch** [comment]
I hope some training creators develop courses for Payload CMS. It's far too rich a platform to learn through trial and error.

**Reply by TimotiusAnrez** [reply]
2nd on this. Payload is great, but lack of learning material makes the learning curve for non CMS experienced developer jarring

**Reply by acandael** [reply]
I fully agree. PayloadCMS could take an example from the Sanity.io learning resources in this regard:

https://www.sanity.io/learn

**Comment by kyle-jorve** [comment]
I keep checking Udemy, as there are some really great instructors on there (Maximilian Schwarzmüller, Jonas Schmedtmann), but so far no luck. @mschwarzmueller and @jonasschmedtmann please halp!

**Comment by shomriddho** [comment]
https://www.youtube.com/watch?v=5Ei3wMPnKss and https://www.youtube.com/watch?v=x4T5V2i_ZaA . this is a small creator
